demos/Makefile

```diff
@@ -22,6 +22,12 @@ $(mc_dir)/free_surface/free_surface.ipynb: $(mc_dir)/base_case/base_case.ipynb
 # dynamic topography requires the checkpoint from the base case
 $(mc_dir)/dynamic_topography/dynamic_topography.ipynb: $(mc_dir)/base_case/base_case.ipynb
 
+# smoothing requires the checkpoint file to be generated
+smoothing/smoothing.ipynb: smoothing/smoothing-example.h5
+
+smoothing/smoothing-example.h5:
+	$(MAKE) -C smoothing smoothing-example.h5
```

Comment:
Unfortunately this won't work either, you already implicitly have this rule https://github.com/g-adopt/g-adopt/blob/fbc272ae21af59ee9c46f0597dbf75d169f7280a/demos/smoothing/Makefile#L7-L13

I think it could be that we're just not doing the explicit LFS fetch... I'll push that to this branch and trigger a render to check it's working.